content/docs/how-we-work/incident-management.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Incident Management
+weight: 15
+description: Managing incidents in production systems is a critical aspect of maintaining system reliability and ensuring continuous service availability. Effective incident management not only focuses on resolving issues as they arise but also on taking steps to prevent future occurrences. Below, we explore the key areas of incident management: Proactive Actions, Corrective Actions, and Preventive Actions.
```

Comment:
@Kalaiselvan88, the colon(:) in the last line of the description _key areas of incident management**:**_causes an error.

Reply:
@Yemaneberhan-Lemma I took this from other files. For e.g. this is present in `accessibility.md` as well. I am wondering how this causes an issue in this file specifically? Might this be any encoding issue?

Reply:
Got it thanks